ZipUtilitiesTests/NOZSwiftTests.swift

```diff
@@ -213,15 +212,14 @@ func TearDownZipQueue()
     }
 
     func testLargeZipDecompress() {
-        var result:NOZDecompressResult? = nil
+        var result:NOZDecompressResult?
         let zipFilePath = try! NOZSwiftDecompressionTests.prepareZipFileForDecompression("Star.Wars.7.Trailer.mp4")
         let expectation = self.expectation(description: "completion closure is called");
         let request = NOZDecompressRequest(sourceFilePath: zipFilePath)
-        let operation = NOZDecompressOperation(request: request, completion: {
-            (op, res) -> Void in
+        let operation = NOZDecompressOperation(request: request) { _, res -> Void in
             result = res
```

Comment:
oh, I really dislike this change.  Is swiftlint really forcing this?  I think it is terrible for legibility to not understand the scope (which the trailing `)` provides)

Reply:
this Swift-y way of doing things is a pretty prevalent style-preference, i think.  it was one of the very earliest things added to the swiftlint rules.

i'm neither a fan/advocate nor detractor; my interest is more in trying to easily enforce the default style-rules as often as possible to avoid (a) getting into the weeds of which style rules to enforce and which not to enforce; and (b) dealing with the extra overhead of workspaces that ingest this having to create whitelist/blacklist exceptions.

i'm going to make one more plea to leave this in; if you still require changes, then in the code that ingests this, i'll endeavor to blacklist the ZipUtilities submodule from this rule if possible, or from all swiftlint style checks if that's the only path.

Reply:
ok, i will take lack of reply for the last 6 days that in this case, ZipUtilities is going to contravene the wider community-preferred default swiftlint style and will blacklist  NOZSwiftTests.swift in our local .swiftlint.yml .

Reply:
Things have been very hectic and this hasn’t been able to get my attention.  It is likely best to not run lint tools on a 3rd party’s source code on general, so I think opting all of ZipUtilities out would be the most straightforward change for the Twitter repo.